package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "src",
+  "name": "warehouse",
```

Comment:
This change happens when running `npm install` outside the container environment.

I tend to use something like `docker compose run --rm static npm ...` for commands to run inside the `static` container.
This also helps avoid any platform-specific dependencies that might be installed as a result - keeping it consistent.

I don't think this is noted anywhere in the dev docs, so apologies! It's been a while since we revamped https://github.com/pypi/warehouse/blob/main/docs/dev/development/frontend.rst - if you wanted to take a stab at that after this, that could be awesome!

Reply:
I removed this change in the rebased branch.